share/new_eds/testing.go

```diff
@@ -67,18 +71,23 @@ func TestStreamer(
 	create createAccessorStreamer,
 	odsSize int,
 ) {
```

Comment:
Why this one is using odsSize still, when other tests are working over eds?

Reply:
Because it is the only test that needs customized EDS with namespaces